### ISSUE TITLE
Add support for 17134.112

### ISSUE THREAD

**Issue by OliverTejada** [stascorp/rdpwrap#516]
Any builds past 17134.1 are unsupported, please add support for this one. Thank you!

Does downgrading to an older version of the terminal services module work? I'm currently in build 17134.112, if I replace termsrv.dll with, say, 16299.XXX could it work on mine?

**Comment by binarymaster** [comment]
Please provide diagnostics screenshot (RDPConf).

**Comment by OliverTejada** [comment]
My bad, I thought it was completely unsupported. I just tried re-installing it online properly and now RDPConf returns fully supported

I had Windows Firewall blocking the outgoing connection the installer makes for some reason
https://i.imgur.com/uM6orI4.png

Tried connecting to one Win10 workstation where I had it installed. It was running Windows 10 1709. Now It is currently operating on build 17134.112 and this update seemed to have broken RDPWrapper  so I thought Microsoft might have fucked something up ever since version 1803 and so I went ahead and tried reinstalling but didn't realize it was unable to fetch the latest .ini from Github due to the firewall blocking it, everything works fine.
